verification/curator-service/api/openapi/openapi.yaml

```diff
@@ -580,6 +580,18 @@ components:
               type: string
         format:
           type: string
+        dateFilter:
+          type: object
+          properties:
+            numDaysBeforeToday:
+              type: number
+              # Must be positive as we subtract it to "today".
```

Comment:
What if we want "only today's data"? Or is that just too risky, such that we wouldn't even allow it?

Reply:
We could change that minimum value to 0 to allow that with the EQ operator but honestly yeah it feels pretty risky.